docs/UserGuide.md

```diff
@@ -1,41 +1,23 @@
 ---
 layout: page
 title: User Guide
----
-
-This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
 
-* Table of Contents
-{:toc}
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `TrAcker.jar` from [here](https://github.com/AY2223S2-CS2103-F11-1/tp).
 
 1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
```

Comment:
Do you mind changing the `AddressBook` here also? Perhaps can replace it with `TrAcker` as well, just to be absolutely sure

Reply:
Ok I've updated it already